can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -11,8 +11,12 @@
 """
 
 import logging
+import os
+import tempfile
 from collections import deque
 
+from filelock import FileLock
```

Comment:
Btw, will the documentation be able to build now? Usually we try to make the modules importable without external packages installed.

Reply:
@christiansandberg I have added a dummy file lock in case `filelock` module is not there